.github/workflows/ci.yml

```diff
@@ -1,34 +1,54 @@
 name: ci
 on: [push, pull_request]
+
 jobs:
   automated-tests:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php_version: ["8.1", "8.2"]
-        composer_command: ["composer install", "composer update --prefer-lowest"]
+        php_version: ["8.1", "8.3"]
+        composer_command: ["composer install"]
     env:
       PHP_VERSION: ${{ matrix.php_version }}
     steps:
-      - name: clone
+      - name: Clone repository
         uses: actions/checkout@v2
-      - run: docker-compose up -d
-      - name: build
+
+      - name: Install Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y docker-compose
+
+      - name: Start services with Docker Compose
+        run: docker-compose up -d
+
+      - name: Build dependencies
         run: docker-compose exec -T php ${{ matrix.composer_command }}
-      - name: test
+
+      - name: Run tests
         run: docker-compose exec -T php ./vendor/bin/phpunit
+
   code-sniffer:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php_version: ["8.1", "8.2"]
+        php_version: ["8.1", "8.3"]
```

Comment:
Why 8.2 is dropped?

Reply:
It is irrelevant to test if we support both 8.1 and 8.3.